src/tools/cgcreate.c

```diff
@@ -63,7 +111,11 @@ int main(int argc, char *argv[])
 	uid_t tuid = CGRULE_INVALID, auid = CGRULE_INVALID;
 	gid_t tgid = CGRULE_INVALID, agid = CGRULE_INVALID;
 
+#ifdef WITH_SYSTEMD
+#endif /* WITH_SYSTEMD */
```

Comment:
`ifdef, endif` here aren't guarding any code.

Reply:
Oops.  It was a wip - I moved them a few lines above to guard the systemd long option names.